src/features/image.rs

```diff
@@ -70,11 +75,14 @@ where
 }
 
 /// Consume an [`image::ImageBuffer`] and return the pixel data as boxed slice.
-fn into_bytes<P>(buffer: image::ImageBuffer<P, Vec<u8>>) -> Box<[u8]>
+fn into_bytes<P, Container>(buffer: image::ImageBuffer<P, Container>) -> Box<[u8]>
 where
 	P: image::Pixel<Subpixel = u8> + image::PixelWithColorType,
+	Container: Deref<Target = [u8]>,
 {
-	buffer.into_raw().into_boxed_slice()
+	// TODO: Specialize this for Vec<u8> to avoid copying when
+	//       https://github.com/rust-lang/rust/issues/31844 lands in stable.
```

Comment:
```suggestion
	// https://github.com/rust-lang/rust/issues/31844 lands in stable.
```

Reply:
Yeah .. thought about that, didn't like it much, but I also don't have a strong opinion about it so sure :p

Reply:
My opinion: too much effort to keep things aligned for little gain. I wouldn't do that in a markdown document either. It might even cause it to be interpreted as code or something ^^